src/utils/cookieConsentManagement.js

```diff
@@ -0,0 +1,58 @@
+import * as Scrivito from "scrivito";
+
+let resolveCookie;
+let resolveCookieConsentCalledBefore = false;
+
+const cookiePromise = new Promise((resolve) => {
+  resolveCookie = resolve;
+});
+
+export function resolveCookieConsent() {
```

Comment:
If I'm not mistaken, this method no longer needed to be exported.

```suggestion
function resolveCookieConsent() {
```